### ISSUE TITLE
[Plugin Request] ec.europa.eu European Commission streamings

### ISSUE THREAD

**Issue by mistnim** [chrippa/livestreamer#993]
During Greece crisis, this website is on fire with press conferences and live updates from Brussels. They stream from here: http://ec.europa.eu/avservices/ebs/live.cfm?page=2&sitelang=en

**Comment by equoia** [comment]
@mistnim I made a quick attempt at a plugin, which should work with the two available live streams.

EbS - http://ec.europa.eu/avservices/ebs/live.cfm?page=1
EbS+ - http://ec.europa.eu/avservices/ebs/live.cfm?page=2

The repo containing the plugin is available here: https://github.com/equoia/livestreamer/tree/plugin-for-EBS-EU-TV

If you are up for it, you can install that version yourself by following the steps below(assuming you got a recent copy of Python and Git):
# Step one.
Open either Terminal or Command Prompt(Depending on which OS you are running, I am just assuming either Windows or OS X)

# Step two.
Enter the following commands

* `git clone git@github.com:equoia/livestreamer.git --branch plugin-for-EBS-EU-TV`
* `cd livestreamer`
* `python setup.py install`

# Step three.
Check if it works.

Try the following command, which should open the EbS Live stream in the best quality available.
* `livestreamer "http://ec.europa.eu/avservices/ebs/live.cfm?page=1&sitelang=en" best`

**Comment by mistnim** [comment]
Thank you, I know those playlists they provide, but I posted this hoping for something better, because they work very badly.

**Comment by equoia** [comment]
Yeah.. I noticed while I was making the plugin, that simply just putting the m3u8 link into VLC will make it play from the very beginning of the stream, while if i used the command `livestreamer hls://[INSERT M3U8_PLAYLIST HERE] best` it would seemingly be streaming live.
